cost-analyzer/values.yaml

```diff
@@ -857,6 +857,13 @@ federatedETL:
 kubecostAdmissionController:
   enabled: false
 
+# Enables or disables the Cost Event Audit pipeline, which tracks recent changes at cluster level
+# and provides an estimated cost impact via the Kubecost Predict API.
+#
+# It is disabled by default to avoid problems in high-scale environments.
+costEventsAudit:
```

Comment:
@dwbrown2 / @kwombach12 - What are your thoughts here? Maybe enabled by default and disabled if it's a problem?

Reply:
Yeah, would propose enabled by default unless there are concerns uncovered during testing.

Reply:
@biancaburtoiu - We may come back here and change this, but let's go ahead and set this to enabled. We'll do our diligence in scale testing this week. If we find anything concerning, we'll revert to this.

Reply:
I've just converted it to default `true`, thanks for the feedback!

Reply:
+1 to enabling by default

Reply:
I do not feel comfortable enabling this by default. We don't have the ability to test this at scale right now effectively since our simulations all scale to number of sets in prometheus, not churn and availability of the kubernetes API. What's more, we should also update this where the env variable is consumed to be a consistent default with what we pick, which I strongly believe should be "false" at this point.

If this doesn't use the kubernetes API via new watchers or saving data out of new watchers, please let me know @biancaburtoiu

Reply:
@AjayTripathy @teevans @biancaburtoiu can you sync to see if we can come up with a scale testing plan you're comfortable with?

Reply:
we cannot come up with a scale testing plan in time for 101.